src/data/nat/factorization.lean

```diff
@@ -166,6 +166,12 @@ lemma prime.factorization_pow {p k : ℕ} (hp : prime p) :
   factorization (p ^ k) = single p k :=
 by simp [hp]
 
+/-- If a product over `n.factorization` doesn't use the multiplicities of the prime factors
+then it's equal to the corresponding product over `n.factors.to_finset` -/
+lemma factorization_rebase_prod {n : ℕ} {β : Type*} [comm_monoid β] (f : ℕ → β) :
```

Comment:
I also don't think this name goes by conventions, but I don't know what would.

Reply:
I've renamed it to `factorization_finset_prod`, which is a little closer to being descriptive